### DISCUSSION TITLE
How to customize logo letters in Logo.tsx

### DISCUSSION THREAD

**Discussion by keybits** [elegantframework/elegant-cli#126] [Q&A] [answered]
I would like to replace the svg paths for the letters 'elegant' at the top left of the screen:

<img width="198" alt="image" src="https://github.com/elegantframework/elegant-cli/assets/809823/33b06ef5-348a-4a3a-a516-7ec650ed98d8">

This is defined in Logo.tsx as individual letters.

Is there a tool I can use to produce svg paths for letters of my own?

**Comment by Bowens20832** [comment, marked as answer]
Hey @keybits, thank you for reaching out 😎 

I created a simple logo using Figma on the free tier, and then exported it as an SVG and copied the code into the Logo.tsx

**Comment by Bowens20832** [comment]
@keybits Just let me know if you need any help or guidance converting your logo to an SVG or anything 😃

**Reply by keybits** [reply]
Thanks, using Figma is fine.
